lib/api/bucketGetTagging.js

```diff
@@ -0,0 +1,94 @@
+const { metadataValidateBucket } = require('../metadata/metadataUtils');
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+const { pushMetric } = require('../utapi/utilities');
+const { errors } = require('arsenal');
+
+//  Sample XML response:
+/*
+    <Tagging>
+        <TagSet>
+            <Tag>
+                <Key>string</Key>
+                <Value>string</Value>
+            </Tag>
+            <Tag>
+                <Key>string</Key>
+                <Value>string</Value>
+            </Tag>
+        </TagSet>
+    </Tagging>
+*/
+
+/**
+ * Convert Versioning Configuration object of a bucket into xml format.
+ * @param {object} tags - set of bucket tags
+ * @return {string} - the converted xml string of the versioning configuration
+ */
+function convertToXml(tags) {
+    const xml = [];
+
+    xml.push('<?xml version="1.0" encoding="UTF-8" standalone="yes"?>',
+             '<Tagging>',
+             '<TagSet>'
+    );
+
+    tags.forEach(tag => {
+        xml.push('<Tag>');
+        xml.push(`<Key>${tag.Key}</Key>`);
```

Comment:
Maybe you can use escapeForXml to avoid xml injection in tag key and value
```suggestion
        xml.push(`<Key>${escapeForXml(tag.Key)}</Key>`);
```

Reply:
done https://github.com/scality/cloudserver/pull/4481/commits/f045bcd4bb3100b1f0412d671d72871a3a920cf1